src/Fantomas/Trivia.fs

```diff
@@ -315,6 +298,12 @@ let private addTriviaToTriviaNode
             findNodeBeforeLineAndColumn triviaNodes range.StartLine range.StartColumn
 
         match nodeBefore, nodeAfter with
+        | Some nb, Some na when
```

Comment:
This made sense to me, does it to you?
I check if the comment is between two nodes on its own line like
```fsharp
let b = zp land (zp)
(* Comment 2 *)
let p = p1 land (b - 1)
```

Reply:
make sense